### DISCUSSION TITLE
FATAL   report error: unable to write results: failed to write results: failed to write with template: template: output template:17:44: executing "output template" at <.MisconfSummary.Successes>: nil pointer evaluating *types.MisconfSummary.Successes

### DISCUSSION THREAD

**Discussion by dverma-cd** [aquasecurity/trivy#5780] [Bugs]
### Description

Using this command . 
trivy image --format template --template "@contrib/junit.tpl"  -o ../junit-report.xml my-rust-app:latest 
Before some hour ago it was working fine . 
i can see there is a new commit in junit.tpl file. 
It was working for me before. 
Commit - https://github.com/aquasecurity/trivy/commit/be5a550491f581311c938f7230b389cb10aacae7 

### Desired Behavior

should generate report without error 

### Actual Behavior

Result is showing but Fatal error 

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Debug Output

```bash
FATAL   report error:
    github.com/aquasecurity/trivy/pkg/commands/artifact.Run
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:455
  - unable to write results:
    github.com/aquasecurity/trivy/pkg/commands/artifact.(*runner).Report
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:285
  - failed to write results:
    github.com/aquasecurity/trivy/pkg/report.Write
        /home/runner/work/trivy/trivy/pkg/report/writer.go:101
  - failed to write with template:
    github.com/aquasecurity/trivy/pkg/report.TemplateWriter.Write
        /home/runner/work/trivy/trivy/pkg/report/template.go:77
  - template: output template:17:44: executing "output template" at <.MisconfSummary.Successes>: nil pointer evaluating *types.MisconfSummary.Successes
```

### Operating System

Macos and ubutu both

### Version

```bash
Commit - https://github.com/aquasecurity/trivy/commit/be5a550491f581311c938f7230b389cb10aacae7
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by jonghyo** [comment]
I also faced same error.

This discussion seams to be similar.
https://github.com/aquasecurity/trivy/discussions/5779

>Only when fetching the previous commit, the scan pass:
https://raw.githubusercontent.com/aquasecurity/trivy/63c33bfa43e758c6907694b2dd255f7da18d10f9/contrib/junit.tpl

Now, we only can pass the trivy scan to use previous ``junit.tpl``.

**Reply by DmitriyLewen** [reply]
We created #5782 to fix this error.
You can get `junit.tpl` from this PR.

**Reply by DmitriyLewen** [reply]
PR has been merged. Take new template from `main` branch.

**Comment by DmitriyLewen** [comment]
Duplicate of #5779
